code/datums/gamemodes/infestation.dm

```diff
@@ -0,0 +1,11 @@
+/datum/game_mode/infestation
```

Comment:
not a fan of this subtyping, maybe just use a signal to attach this behaviour.
crash and distress gamemodes share basically no code.

Reply:
A lot of stuff is at the base of  `/datum/game_mode` which is not great when we plan to allow for entirely different game modes. I just didn't move it all there because it would bloat the PR quite a bit, and it's big enough already. Still, there's a few things that already use it, and there's another open PR to make use of it.
Xenos being the main antags is quite specific behavior.